compose/cmd.sh

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env bash
```

Comment:
Suggestion: add a comment describing what this script is meant for?
Also perhaps rename it to something more descriptive, eg. `cardano-node-wrapper.sh` or something like that?